lib/src/st2110/st_tx_video_session.c

```diff
@@ -2329,6 +2325,7 @@ static int tv_init_hw(struct mtl_main_impl* impl, struct st_tx_video_sessions_mg
     flow.bytes_per_sec = tv_rl_bps(s);
     mtl_memcpy(&flow.dip_addr, &s->ops.dip_addr[i], MTL_IP_ADDR_LEN);
     flow.dst_port = s->ops.udp_port[i];
+    if (ST21_TX_PACING_WAY_TSN == s->pacing_way[i]) flow.launch_time_enabled = true;
```

Comment:
Request the launch time feature.

Reply:
Can you remove `if (ST21_TX_PACING_WAY_TSN == s->pacing_way[i])`, launch_time_enabled is a general attribute for st2110-20 flow.

Reply:
launch_time_enabled is NOT always true for ST 2110-20 flow. 

Only TSN pacing enabled, launch_time_enabled is true for requesting launch time feature. 

Others pacing mode, launch_time_enabled is false for no launch time feature requirement.